streamlit_web.py

```diff
@@ -120,8 +128,7 @@ def retreive_data(data_retreive):
         except TypeError:
             st.sidebar.warning(
                 "No data imported. Please check the reflection document input")
-            with open("README.md") as readme_file:
-                st.markdown(readme_file.read())
+            landing_pg()
```

Comment:
Instead of `landing_pg()` and creating a duplicate selectbox, just call `readme()`. The same with the following couple of lines.

Reply:
Thanks for this comment, I didn't realize it would create this duplicate sidebar. I believe the changes have been fixed with my most recent push.